contracts/axelar-gas-service/tests/test.rs

```diff
@@ -181,6 +181,91 @@ fn pay_gas_for_contract_call() {
     );
 }
 
+#[test]
+fn fail_add_gas_zero_gas_amount() {
+    let (env, contract_id, _gas_collector, client) = setup_env();
+
+    let asset = env.register_stellar_asset_contract_v2(Address::generate(&env));
+    let sender: Address = Address::generate(&env);
+    let message_id = String::from_str(
+        &env,
+        &format!(
+            "{}-{}",
+            "0xfded3f55dec47250a52a8c0bb7038e72fa6ffaae33562f77cd2b629ef7fd424d", 0
+        ),
```

Comment:
would be cleaner to just hardcode the string instead of using `format!()`. 

Also, im just curious how you got to this message id, was it random?

Reply:
Yeah, it's just a random value, I grabbed it from the unit test `fn refund()` in this file. I think the formatting is intended to make it clear that we are explicitly combining the tx hash and index. [In the EVM implementation](https://github.com/axelarnetwork/axelar-cgp-solidity/blob/b323ae613a4286e4849bbf81dd8922e80fe4fdca/contracts/gas-service/AxelarGasService.sol#L329), they were separated, but in Stellar, we combined them into message_id, so I guess we want to explicitly test using hashtx + index.